scala/apps/dwd/src/Main.scala

```diff
@@ -15,247 +15,331 @@ import semagrams.Background
 import semagrams.EntityCollection
 
 import scala.language.implicitConversions
+import scala.reflect.ClassTag
```

Comment:
I'm just using it so that I can print out the type of an attribute. I would love to have a better way to access the target type information of an Attr, but scala doesn't make it so easy to work with the types.